recipes/toil/meta.yaml

```diff
@@ -0,0 +1,124 @@
+# copied from https://github.com/bioconda/bioconda-recipes/blob/9c08a16bac9b7d0a076578823d5008b8bbfc466e/recipes/toil/meta.yaml
```

Comment:
@mbargull how is the addition of bioconda packages into conda-forge these days? This one is pure Python and seems like a good candidate to be here.

Reply:
@conda-forge/bioconda-recipes : Hi folks, can we get some thoughts on adding this recipe to conda-forge?